Sources/CMakeLists.txt

```diff
@@ -12,6 +12,31 @@
 ##
 ##===----------------------------------------------------------------------===##
 
+if(NOT SwiftFoudation_MACRO)
+  include(ExternalProject)
+  # The macros are required for building Swift-Foundation. Build them for the
+  # build machine.
+  ExternalProject_Add(FoundationMacros
+    SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/FoundationMacros"
+    INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/FoundationMacros"
+    CMAKE_ARGS
+      -DCMAKE_BUILD_TYPE=Release
+      -DCMAKE_Swift_COMPILER=${CMAKE_Swift_COMPILER}
+      -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
```

Comment:
I haven't seen the `<INSTALL_DIR>` syntax before, out of curiosity, what does it mean?

Reply:
This is a special placeholder that will be filled in with the install path. This is documented in the `ExternalProject` documentation IIRC.